go-controller/pkg/clustermanager/userdefinednetwork/controller_test.go

```diff
@@ -84,9 +84,9 @@ var _ = Describe("User Defined Network Controller", func() {
 					Expect(err).NotTo(HaveOccurred())
 					return normalizeConditions(udn.Status.Conditions)
 				}).Should(Equal([]metav1.Condition{{
-					Type:    "NetworkReady",
+					Type:    "NetworkCreated",
```

Comment:
Did you mean for a const in controller_test.go, used in unit-tests only?

Please note that  since the controller.go  and controller_test.go defined under the same package (`controller`), 
if the const is added to controller_test.go nothing prevents the next dev use it in controller.go implicitly. 

Having the condition type hardcoded in tests has a benefit, changing it will not go silently as tests will break, and the reviewer will have clear indication a condition type was changed.
If the code and unit-tests are separated to different package (controller_test.go under `controller_test`) we could have an additional const in controller_test.go for convenience.

Reply:
It make sense, but note that if we use this const in tests, and it value changes, unit-tests will not fail and we lose this indication as tests will pass.
Having it hardcoded in tests, ensure CI will be red on such changes.

Reply:
you can leave unit tests with string if you find it better, but I would add a constant at least in the controller code itself, and ensure udn and cudn use the same condition

Reply:
Done, PTAL